### ISSUE TITLE
Come up with a better icon

### ISSUE THREAD

**Issue by phaistonian** [phaistonian/Chrome-Notepad#31]
Send by a Chrome Notepad's user.

![image](https://user-images.githubusercontent.com/796530/67737365-89849180-fa13-11e9-8f5d-55e96b50800d.png)

**Comment by gs-akhan** [comment]
Yes, I do have a new icon, I will change and commit it this weekend.  :)
